src/startupscriptgenerator/dotnetcore/scriptgenerator.go

```diff
@@ -36,7 +36,7 @@ func (gen *DotnetCoreStartupScriptGenerator) GenerateEntrypointScript(scriptBuil
 	defaultAppFileDir := filepath.Dir(gen.DefaultAppFilePath)
 
 	scriptBuilder.WriteString("readonly appPath=\"" + gen.RunFromPath + "\"\n")
-	scriptBuilder.WriteString("userStartUpCommand=\"" + gen.UserStartupCommand + "\"\n")
+	scriptBuilder.WriteString("userStartUpCommand=(" + gen.UserStartupCommand + ")\n")
```

Comment:
@gpcastro I found this bug while writing tests for the mutli-runtimeconfig.json files. Basically a startup command like `dotnet foo.dll` does not work currently 😞 as it is being considered as a single value(so the above change is to make as an array of values instead) and we check if a file with that name exists or not. 
A workaround for this would be to create a script file(which does `dotnet foo.dll` in it and drop it in the app output folder and use this script as a startup command.
I had tests for the user startup command scenario but we have a fallback logic where we try to figure out the command (using runtimeconfig.json hueristics) and so the tests were always able to find the file successfully.